test/test_prototype_builtin_datasets.py

```diff
@@ -107,29 +107,15 @@ def test_transformable(self, test_home, dataset_mock, config):
 
         next(iter(dataset.map(transforms.Identity())))
 
-    @parametrize_dataset_mocks(
-        DATASET_MOCKS,
-        marks={
-            "cub200": pytest.mark.xfail(
-                reason="See https://github.com/pytorch/vision/pull/5187#issuecomment-1015479165"
-            )
```

Comment:
I'm guessing this was fixed by pytorch/pytorch#73118. Can you confirm @NivekT?

Reply:
I believe by enabling the serialization of `fork`, it should fix one of the tests but I'm not sure if it is that one specifically. Does any step if that test use `fork`?

Reply:
Also worth mentioning that this other PR also fixed serialization related problems https://github.com/pytorch/pytorch/pull/72896

Reply:
This failed before because the `Demultiplexer` internally used a generator and thus was unable to be pickled after the iteration started.